modules/liquidity-pools/base/src/lib.rs

```diff
@@ -117,6 +128,38 @@ decl_module! {
 			Self::_withdraw_liquidity(&who, pool_id, amount)?;
 			Self::deposit_event(RawEvent::WithdrawLiquidity(who, pool_id, amount));
 		}
+
+		#[weight = 10_000]
+		pub fn set_identity(origin, #[compact] pool_id: LiquidityPoolId, identity_info: IdentityInfo) {
```

Comment:
Implementation in `decl_module` cannot neither format nor get hints from IDE, please extract the code into a private method like other dispatchable calls.